### PR TITLE
docs: update `ADOPTERS` in README.md

### PR DESCRIPTION
**Pull request by ShivangShandilya**
This PR fixes: #9715 

ADOPTERS now point to - https://github.com/keptn/community/blob/main/ADOPTERS

### CODE REVIEW

**Comment by RealAnna** [comment]
Hi @ShivangShandilya thankyou for the contribution! We do have a semantic check in place where if there is a single commit that should follow  [conventional commit guidelines](https://www.conventionalcommits.org/en/v1.0.0/) since it willbe used in the changelog. Can you fix the commit message or push a second empty commit?

**Review by mowies — DISMISSED** [review]

**Comment by ShivangShandilya** [comment]
> Hi @ShivangShandilya thankyou for the contribution! We do have a semantic check in place where if there is a single commit that should follow [conventional commit guidelines](https://www.conventionalcommits.org/en/v1.0.0/) since it will be used in the changelog. Can you fix the commit message or push a second empty commit?

I made an empty commit but looks like my DCO check got removed. Don't know why that happened but will take care of semantic check in my future contributions.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ an overview of user stories that are currently in the focus of development for t
 3 years of hard work will soon reach a long awaited milestone: [Keptn 1.0 with LTS (Long Time Support)](https://docs.google.com/document/d/1RdFegnZrxjWxJAem9auaeVQ5_mKl5wFlwd6MgF1ot0s/edit#heading=h.qoctq8iujkhs) brings you automated release validation based on SLOs that you can easily integrate into your existing DevOps Tools (deployment, test and observability).
 
 If you want to explore Keptn then keep scrolling down to get all information!
-
+
```
Comment (mowies) [inline README.md:28]:
```suggestion

```

**Comment by mowies** [comment]
@ShivangShandilya please fix the  DCO check by running the following commands:
```
git pull
git rebase HEAD~3 --signoff
git push --force-with-lease origin adopters
```

**Comment by ShivangShandilya** [comment]
> @ShivangShandilya please fix the DCO check by running the following commands:
> 
> ```
> git pull
> git rebase HEAD~3 --signoff
> git push --force-with-lease origin adopters
> ```

done thanks @mowies

**Review by mowies — APPROVED** [review]

**Review by RealAnna — APPROVED** [review]

**Pull request merged**